Map update

kajtek14102 opened the pull request:
This update is a preparation for right click functionality. Will not change anything in game right now.
Attached the (shitty) script that I used to generate those - can be used to maintain the lists in future. Will try to find time and make it more readable later.

Also I let myself delete from map cards that are not in game (mostly promoted ones) as they were only making it more confusing

kajtek14102 committed:
Not used cards removed from Map
Empty linked cards list aded to all cards
LinkedCards added to GwentCard and Map

kajtek14102 committed:
linked cards not dependent on tags

kajtek14102 committed:
added linked cards for ones with more linked
added a script to generate those tags (doesnt change anything and is not called by refresh)

kajtek14102 commented:
So I have no idea why all checks just failed immediately. I can assure that it didn't break anything locally. If anybody has any suggestion what might have caused it please share them

neal2018 commented:
> delete from map cards that are not in game

Please do not delete or reorder the entry in GwentMap, because the deck code relies on the order in GwentMap. (I know it is a bad design, and we maybe need to redesign deck code better)

kajtek14102 commented:
> > delete from map cards that are not in game
> 
> Please do not delete or reorder the entry in GwentMap, because the deck code relies on the order in GwentMap. (I know it is a bad design, and we maybe need to redesign deck code better)

Didn't know that. Will put all not used cards back when I find a time for it

kajtek14102 committed:
deleted cards brought back

kajtek14102 commented:
> > delete from map cards that are not in game
> 
> Please do not delete or reorder the entry in GwentMap, because the deck code relies on the order in GwentMap. (I know it is a bad design, and we maybe need to redesign deck code better)

Brought those cards back. Should be ok now

kajtek14102 commented:
Let me know if there is anything more to change. I would be great for me if you could merge this as I would like to have this data for testing in other branch

kajtek14102 committed:
Merge branch 'diy' into MapUpdate

neal2018 commented:
I will deploy this after tournament to prevent possible server break

kajtek14102 committed:
Fixed to not display agents.
Added Uma
Fixen nilgard speling in code (so nilgard dependent filtering now works)

kajtek14102 committed:
Merge branch 'MapUpdate' of https://github.com/kajtek14102/LegacyGwent into MapUpdate

neal2018 committed:
bump GwentMap version
